src/components/picker/api/picker.api.json

```diff
@@ -0,0 +1,97 @@
+{
+  "name": "Picker",
+  "category": "form",
+  "description": "Picker Component, support single or multiple selection, blurModel and nativePicker",
+  "extends": ["TextField"],
+  "modifiers": ["margin", "padding", "position"],
+  "example": "https://github.com/wix/react-native-ui-lib/blob/master/demo/src/screens/componentScreens/PickerScreen.js",
+  "images": [
+    "https://github.com/wix/react-native-ui-lib/blob/master/demo/showcase/Picker/Default.gif?raw=true",
+    "https://github.com/wix/react-native-ui-lib/blob/master/demo/showcase/Picker/MultiPicker.gif?raw=true",
+    "https://github.com/wix/react-native-ui-lib/blob/master/demo/showcase/Picker/NativePicker.gif?raw=true",
+    "https://github.com/wix/react-native-ui-lib/blob/master/demo/showcase/Picker/DialogPicker.gif?raw=true",
+    "https://github.com/wix/react-native-ui-lib/blob/master/demo/showcase/Picker/CustomPicker.gif?raw=true"
+  ],
+  "note": "'useNativePicker' prop requires installing the '@react-native-picker/picker' native library",
```

Comment:
Is that still relevant? 
Aren't we using our own implementation of WheelPicker from Incubator?

Reply:
Yes, don't we want to remove the `lib/WheelPicker` (and this dependency)? Should I add it to V7?

Reply:
Better do it officially in v7, yes

Reply:
Added a row to V7